changelogs/unreleased/gh-7089-applier-vclock-race.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/replication
+
+* Fix a case when replication was broken on master side with
+  `ER_INVALID_MSGPACK` (gh-7089).
```

Comment:
Nit: Please, use past tense: 'fixed', 'updated' and so on. See https://www.tarantool.io/en/doc/latest/contributing/release_notes/.

Reply:
Done